documentation/Semantics.md

```diff
@@ -1,206 +1,139 @@
 <!--
-Copyright (c) 2018, NVIDIA CORPORATION.  All rights reserved.
+Copyright (c) 2018-2019, NVIDIA CORPORATION.  All rights reserved.
 -->
 
-The semantic pass will determine whether the input program is a legal Fortran
-program.
+# Semantic Analysis
+
+The semantic analysis pass will take a parse tree for a syntactically
```

Comment:
This sentence uses future tense, and the rest of the document uses present tense.  How about "The semantic pass determintes"?